cli/data/get.go

```diff
@@ -111,17 +112,26 @@ func copyPathToDestination(source S3Path, destination string, keys []string, sub
 }
 
 func readHEAD(session *session.Session, source S3Path) string {
-	svc := s3.New(session)
+	tempFile, err := ioutil.TempFile("", "HEAD")
+	if err != nil {
+		exitErrorf("Unable to create temp file: %v", err)
+	}
 
-	out, err := getObject(svc, aws.String(source.bucket), aws.String(source.path))
+	defer os.Remove(tempFile.Name())
 
+	svc := s3.New(session)
+
+	err = copyS3ObjectToFile(svc, source, source.path, tempFile)
 	if err != nil {
 		exitErrorf("Error reading HEAD: %v", err)
```

Comment:
Could maybe change to `Error copying` now? Though it does look like the error would still come from read. Just as a way to get to isolate the errors returned really.

Reply:
Good point, makes it easier to distinguish the different exit points in this method.  Done